compiler/front/msgs.nim

```diff
@@ -600,18 +606,20 @@ proc report*(conf: ConfigRef, node: PNode): TErrorHandling =
   assert node.kind == nkError
   return conf.report(conf.astDiagToLegacyReport(conf, node.diag))
 
+proc fillReportAndHandleVmReport(c: ConfigRef, r: var Report, reportFrom: InstantiationInfo) =
```

Comment:
```suggestion
proc fillReportAndHandleVmTrace(c: ConfigRef, r: var Report,
                                reportFrom: InstantiationInfo) =
```
The procedure doesn't handle VM reports in general, only the associated stack-trace report.